src/views/Home/components/EarnAPYCard.tsx

```diff
@@ -18,10 +18,10 @@ const StyledFarmStakingCard = styled(Card)`
     max-width: none;
   }
 `
-const Label = styled(Heading)`
+const Label = styled(Heading).attrs({ size: 'lg' })`
   font-size: 24px;
 `
-const CardMidContent = styled(Heading)`
+const CardMidContent = styled(Heading).attrs({ size: 'xl' })`
```

Comment:
The `size` prop determines the font size of the component. So you don't need this component but can use the Heading component directly

```
<Heading size="lg" />
```